src/Courses/TopicGrades/GradingPage.tsx

```diff
@@ -38,6 +39,17 @@ export const TopicGradingPage: React.FC<TopicGradingPageProps> = () => {
         grade?: StudentGrade,
         gradeInstance?: StudentGradeInstance,
     }>({});
+    const { url } = useRouteMatch();
+    const history = useHistory();
+
+    useEffect(() => {
+        const queryString = qs.stringify(_({
```

Comment:
Does this trigger a re render from react routers?

Reply:
I haven't looked specifically if it causes rerenders higher up, but the page exhibits no stuttering.